src/Civi/Buildkit/Phars.php

```diff
@@ -100,11 +100,12 @@ protected static function downloadFile(string $remoteUrl, string $expectHash, st
       throw new \Exception(sprintf("Download for \"%s\" failed SHA-256 check (expect=%s, actual=%s)", $remoteUrl, $expectHash, $actualHash));
     }
 
-    if (file_exists($localFile)) {
+    // Civix could be a directory if checked out as a repo (unusual but possible configuration).
```

Comment:
So you're file-layout is like this?

```
BUILDKIT/bin
BUILDKIT/bin/amp
BUILDKIT/bin/civix/bin/civix
```

That seems like a weird place to put the folder.

But, OK, if you're skip it... then shouldn't it skip the entire download? Otherwise, you're just gonna keep re-running 2mb HTTP requests and never using the content.

Reply:
I'll do an update for ^^

Reply:
Well I don't call download-tools very often.....

Reply:
@eileenmcnaughton OK, try this:  https://github.com/civicrm/civicrm-buildkit/pull/809